core/src/main/java/com/amplifyframework/devmenu/DevMenuFileIssueFragment.java

```diff
@@ -28,6 +32,18 @@
  * file issue view for the developer menu.
  */
 public final class DevMenuFileIssueFragment extends Fragment {
+    // Minimum number of characters required for the issue description.
+    private static final int MIN_DESCRIPTION_LENGTH = 140;
```

Comment:
If this will actually _prevent_ upload, we may want to bring it down to like 50. We can always triage/delete crappy issues, once we get them. But I could see myself getting frustrated if I had something really simple to share, but needed to pad it out to 140.

Reply:
I think 140 characters is a reasonable requirement to help make sure the customer is actually providing a description of the issue. For reference, the following description (which still seems pretty short) is 148 characters:
"Some description of the issue would be added here. This description meets the minimum character count requirement. More information about the issue."

Reply:
Okay, cool! Let's run with it.